exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -134,20 +134,36 @@ export default ExpFrameBaseComponent.extend({
   }),
 
   // Represent the sorted cards in a human-readable format for storage in the database
-  responses: Ember.computed('cardSortResponse', function () {
-      let cardSortResponse = this.get('cardSortResponse');
-
-      // Final data should be returned as object {categoryName: [cardIdentifiers] }
-      let responses = {};
-
-      // Assumption: this unpacks a list of { categories: {name: name, cards: [cards]} } objects
-      for (let categorySet of cardSortResponse) {
-          for (let category of categorySet.categories) {
-              let name = category.name.split('.').pop();
-              responses[name] = category.cards.map((cardItem) => cardItem.id);
+  responses: Ember.computed('cardSortResponse', 'cardSortResponse2', {
+      get(key) {
+          // Final data for each card-sort should be returned as object {categoryName: [cardIdentifiers] }
+          let responses = {
+              cardSort1: {},
+              cardSort2: {}
+          };
+          let cardSortResponse;
+          if (this.get('cardSortResponse')) {
+              cardSortResponse = this.get('cardSortResponse');
+              for (let category of cardSortResponse) {
+                  let name = category.name.split('.').pop();
+                  responses['cardSort1'][name] = category.cards.map((cardItem) => cardItem.id);
+              }
+          }
+          if (this.get('cardSortResponse2')) {
+              cardSortResponse = this.get('cardSortResponse2');
+              // Assumption: this unpacks a list of { categories: {name: name, cards: [cards]} } objects
+              for (let categorySet of cardSortResponse) {
+                  for (let category of categorySet.categories) {
+                      let name = category.name.split('.').pop();
+                      responses['cardSort2'][name] = category.cards.map((cardItem) => cardItem.id);
+                  }
+              }
           }
+          return responses;
+      },
+      set(key, value) {
```

Comment:
Interesting question!

I recall you mentioning that ISP is a project that asks users to respond several times in one day. Exactly how long do you expect to go between sessions? (do we want to support completing the form over several visits?)

If we expect a quick return, resuming at the last page makes a lot of sense. If they spend several days between visits, we may just want them to start at the beginning.